ethanschreur/kids-krafts
# Sellers can edit purchases

**What does this PR do?**
- This pull request allows sellers to edit customer purchases

**Description of the task to be completed** 
- Add a save button to the edit order form
- Make that save button update the orders information
- Pressing the save button should take you back to the orders page

**How should this  be tested**
- (On Windows) Run `. venv/scripts/activate` 
- `python run.py`
- Go to http://127.0.0.1:5000/orders (log in if you have to)
- Fill out the "Add Orders" form and enter it
- Press on that order in the orders table
- Fill out the "Add a Purchase" form
- Edit that purchase's information in the "Edit Purchases" section
- Press the save button
- Confirm that that purchase's information was updated
